RecoEgamma/PhotonIdentification/src/PhotonMIPHaloTagger.cc

```diff
@@ -290,17 +290,11 @@ std::vector<double> PhotonMIPHaloTagger::GetMipTrailFit(const reco::Photon* phot
            
 
         //Lets Initialize them first for each iteration 
-        res    = 0.0;
-        res_sq = 0.0;
-        wt     = 0.0; 
```

Comment:
@VinInn that is indeed a fair point since there are other variables that are initialized similarly and the static analyzer gives no warnings about them.